arch/arm/src/imx6/imx_enet.c

```diff
@@ -102,6 +102,10 @@
 #  error "Need at least one TX buffer"
 #endif
 
+#if !defined (CONFIG_NET_TCP_WRITE_BUFFERS) && (CONFIG_IMX_ENET_NTXBUFFERS != 1)
+#  error "CONFIG_IMX_ENET_NTXBUFFERS must be 1 without TCP_WRITE_BUFFERS"
```

Comment:
error "TCP_WRITE_BUFFERS are disabled. CONFIG_IMX_ENET_NTXBUFFERS must be set to 1"

Reply:
@davids5 

Thanks! I'll send a new PR later.